apps/frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -1,38 +1,44 @@
 import { Icon } from "~/libs/components/components.js";
+import { IconSize } from "~/libs/enums/icon-size.enum.js";
```

Comment:
Should be imported from` ~/libs/enums/enums.js`, I guess

Reply:
Done